pkg/imgpkg/bundle/bundle_images_lock.go

```diff
@@ -15,6 +15,64 @@ import (
 	"github.com/k14s/imgpkg/pkg/imgpkg/lockconfig"
 )
 
+func (o *Bundle) AllImagesLock() (*ImagesLock, error) {
+	imgsLock, err := o.buildAllImagesLock(map[string]bool{})
+	if err != nil {
+		return nil, err
+	}
+
+	imgsLock.AddImageRef(lockconfig.ImageRef{
```

Comment:
i dont think we need this because of https://github.com/vmware-tanzu/carvel-imgpkg/pull/99/files#diff-fae76c4e2e5ddd7698236747d7211e5725cf904a59c4bcd081555471458d80f8R170